ci/symfony.yml

```diff
@@ -0,0 +1,28 @@
+name: Symfony
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  symfony-tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: shivammathur/setup-php@15c43e89cdef867065b0213be354c2841860869e
```

Comment:
I use same template as Laravel, i edit this today.

Reply:
Just saw that. Looks like it was added by the action author, @shivammathur himself.

@shivammathur can we use official version instead of commits now?

Reply:
For 3rd party actions, it's preferred to pin to a SHA for security reasons. If users want to pin to a version, they can modify their workflow.

Reply:
>We require that Actions outside of the actions organization be pinned to a specific SHA.
https://github.com/actions/starter-workflows/blob/main/CONTRIBUTING.md#previous-guidelines-for-new-starter-workflows

Reply:
@NinadKavimandan @TheGarious
Please use this commit hash `2cb9b829437ee246e9b3cac53555a39208ca6d28` for `2.13.0` release of setup-php.

You can add a comment to use `v2` and with a link to the [versioning ](https://github.com/shivammathur/setup-php#bookmark-versioning)policy of setup-php similar to the [ruby starter workflow](https://github.com/actions/starter-workflows/blob/main/ci/ruby.yml#L27).

Reply:
I add your comment
@shivammathur  @joshmgross